File: saizad-mvvm/src/main/java/com/saizad/mvvm/delegation/fragment/FragmentCB.java
package com.saizad.mvvm.delegation.fragment;

import androidx.annotation.LayoutRes;

import com.saizad.mvvm.components.SaizadBaseViewModel;
import com.saizad.mvvm.delegation.BaseCB;

public interface FragmentCB<V extends SaizadBaseViewModel> extends BaseCB<V> {

    @LayoutRes
    int layoutRes();

    boolean persistView();
}

File: saizad-mvvm/src/main/java/com/saizad/mvvm/delegation/fragment/FragmentCB.kt
package com.saizad.mvvm.delegation.fragment

import androidx.annotation.LayoutRes
import com.saizad.mvvm.components.SaizadBaseViewModel
import com.saizad.mvvm.delegation.BaseCB

interface FragmentCB<V : SaizadBaseViewModel> : BaseCB<V> {
    @LayoutRes
    fun layoutRes(): Int
    fun persistView(): Boolean
}